presto-main/src/main/java/io/prestosql/execution/buffer/PagesSerdeUtil.java

```diff
@@ -60,6 +70,16 @@ public static void writeSerializedPage(SliceOutput output, SerializedPage page)
         output.writeBytes(page.getSlice());
     }
 
+    private static void updateChecksum(XxHash64 hash, SerializedPage page)
```

Comment:
nit: aren't we typically ordering methods that usage is first and declaration follows?

Reply:
`updateChecksum` is placed right under `writeSerializedPage`, because they need to stay in sync.
i added a code comment